src/backoffice/settings/languages/workspace/language-root/language-root-table-delete-column-layout.element.ts

```diff
@@ -0,0 +1,70 @@
+import { UUITextStyles } from '@umbraco-ui/uui-css';
+import { css, html, nothing } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { UmbLanguageStore, UMB_LANGUAGE_STORE_CONTEXT_TOKEN } from '../../language.store';
+import type { LanguageDetails } from '@umbraco-cms/models';
+import { UmbLitElement } from '@umbraco-cms/element';
+import { UmbModalService, UMB_MODAL_SERVICE_CONTEXT_TOKEN } from 'src/core/modal';
+
+@customElement('umb-language-root-table-delete-column-layout')
+export class UmbLanguageRootTableDeleteColumnLayoutElement extends UmbLitElement {
+	static styles = [UUITextStyles, css``];
+
+	@property({ attribute: false })
+	value!: LanguageDetails;
+
+	private _languageStore?: UmbLanguageStore;
+	private _modalService?: UmbModalService;
+
+	constructor() {
+		super();
+		this.consumeContext(UMB_LANGUAGE_STORE_CONTEXT_TOKEN, (instance) => {
+			this._languageStore = instance;
+		});
+
+		this.consumeContext(UMB_MODAL_SERVICE_CONTEXT_TOKEN, (instance) => {
+			this._modalService = instance;
+		});
+	}
+
+	private _handleDelete(event: MouseEvent) {
+		event.stopImmediatePropagation();
+		if (!this._languageStore) return;
+
+		const modalHandler = this._modalService?.confirm({
```

Comment:
Should we consider making a more generic 'confirm' dialog with an optional parameter to customise the text?